doc/modules/mod_push_service_mongoosepush.md

```diff
@@ -12,10 +12,8 @@ The following `publish-options` that are added to the hook are directly passed t
   E.g. `fcm`, `apns`.
  * `device_id` - has to be specified and the value must be valid device token received from push notification service provider specified in `service` option
  
-In addition to those `publish-options` you may specify also `silent` option, that when set to 
-`true` will result in "silent" notification. 
-Silent notifications send only data payload to push notifications service provider with all 
-fields specified in the notification without any modification and/or filtering.
+In addition to those `publish-options` you may specify also `silent` option, that when set to `true` will result in "silent" notification. 
+Silent notifications send only data payload to push notifications service provider with all fields specified in the notification without any modification and/or filtering.
```

Comment:
Silent notifications send only the data payload to the push notification service provider with all fields specified in the notification without any modification and/or filtering.